src/UI/Component/Icon/Factory.php

```diff
@@ -9,14 +9,30 @@
 	 * ---
 	 * description:
 	 *   purpose: >
-	 *    Standard Icons represent ILIAS Objects.
-	 *   composition: >
+	 *    Standard Icons represent ILIAS Objects. In their outlined version, they are used to be layered upon a picture or
+     *    if the Standard Icon should not too much of the users attention.
+     *   composition: >
```

Comment:
sry, something seems of here with the intends. however, the yaml format seems intact. Ill correct that.